contrib/podman/files/registries.conf

```diff
@@ -0,0 +1,79 @@
+# For more information on this configuration file, see containers-registries.conf(5).
```

Comment:
this should be part of a containers-common instead, like in https://github.com/void-linux/void-packages/blob/314341f80a7eeb430d4b4efe01e733d8873cf22c/srcpkgs/containers-common/template or https://git.alpinelinux.org/aports/tree/community/containers-common/APKBUILD , since there's a bunch more configs too (and so you don't need to manually specify this config at all)

Reply:
preferably more like the latter tho (also how archlinux does it) since you have to match up the versions of a few repos for all the files and check them against the podman version